tests/factories/user_item_factories.py

```diff
@@ -0,0 +1,23 @@
+from tests.factories import *
+
+from sourcemash.models import User, UserItem
+from user_factories import UserFactory
+from item_factories import ItemFactory
+
+
+class UserItemFactory(SQLAlchemyModelFactory):
+    FACTORY_FOR = UserItem
+    FACTORY_SESSION = db.session
+
+    user = factory.SubFactory(UserFactory)
+    item = factory.SubFactory(ItemFactory)
+
+
+class UserItemUpvoteFactory(UserItemFactory):
+    vote = 1
+    item = factory.SubFactory(ItemFactory, totalVotes=1)
```

Comment:
Try something like this? http://factoryboy.readthedocs.org/en/latest/reference.html#postgenerationmethodcall